ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1550,6 +1588,62 @@ static void LogError(const char* formatString, ...)
     vasprintf(&logString, formatString, dataArgs);
     va_end(dataArgs);
     
-    s_callbacks.LogError(logString);
+    if (s_callbacks.LogError != nullptr)
```

Comment:
In testing I found that s_callbacks.LogError could be null depending on if the mount process is fully set up.  Instead of crashing and getting no error, I switched this to not crash and give an error in debug mode.